platforms/intellij/compose/playground/build.gradle.kts

```diff
@@ -14,12 +14,18 @@
  * limitations under the License.
  */
 import org.jetbrains.kotlin.gradle.plugin.KotlinPlatformType
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
   alias(libs.plugins.kotlin.multiplatform)
   alias(libs.plugins.compose)
   alias(libs.plugins.kotlin.plugin.compose)
   alias(libs.plugins.lint)
+  alias(libs.plugins.roborazzi)
+}
+
+roborazzi {
+  outputDir = file("src/jvmTest/kotlin/foundry/intellij/compose/playground/snapshots/images")
```

Comment:
Is there a property input of this? We should ideally use this in gradle for lazier eval

```kotlin
outputDir.set(layout.projectDirectory.dir("src/jvmTest/kotlin/foundry/intellij/compose/playground/snapshots/images"))
```

if `outputDir` is just a `File` and not a `Property<File>`, then you can still use the above but just write `.asFile.get()` at the end


```kotlin
outputDir = layout.projectDirectory.dir("src/jvmTest/kotlin/foundry/intellij/compose/playground/snapshots/images").asFile.get()
```